test/startservers.py

```diff
@@ -16,6 +16,8 @@
 
 # Keep these ports in sync with consul/config.hcl
 SERVICES = (
+    # TODO(#5294) After we have separate RVA and VA binaries deployed, remove
+	# the old boulder-remoteva-a/b and rename boulder-remoteva2-a/b.
```

Comment:
This line uses a tab, rather than spaces, for comment alignment.

Reply:
Weird, fixed.